clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -7042,12 +7098,44 @@ void SPIRVTranslator::ConstructJob(Compilation &C, const JobAction &JA,
     TranslatorArgs.push_back("-spirv-ext=+all");
   }
   for (auto I : Inputs) {
+    if (I.getType() == types::TY_TempIRfilelist) {
+      std::string FileName(I.getFilename());
```

Comment:
Suggest initializing before the condition and reusing at line 7110

Reply:
Okay.